_el/exercises.md

```diff
@@ -336,172 +204,52 @@ piece of code with the output it produces".
+
+Οι [[Srid2016](#CITE)] υιοθέτησαν μια διαφορετική προσέγγιση. Χρησιμοποίησαν [ασαφείς δοκιμές](#g:fuzz-testing) (δηλ., τυχαία παραγόμενες δοκιμασίες) για να ελέγξουν εαν ο κώδικας του μαθητή κάνει το ίδιο πράγμα με μια πρότυπη υλοποίηση που παρέχεται από τον εκπαιδευτή. Στην πρώτη εργασία ενός εισαγωγικού μαθήματος με 1400 μαθητες, οι ασαφείς δοκιμές μπόρεσαν να εντοπίσουν λάθη τα οποία ξέφυγαν από μια σειρά δοκιμαστικών περιπτώσεων που έγιναν χειρόγραφα για περισσότερο από 48% των μαθητών, γεγονός που δείχνει σαφώς την αξία του.
+
+Οι [[Basu2015](#CITE)] έδωσαν στους εκπαιδευόμενους μια σειρά από λύσεις δοκιμαστικών τεστ, αλλά οι μαθητές έπρεπε να ξεκλειδώσουν το καθένα απαντώντας σε ερωτήσεις σχετικά με την αναμενόμενη συμπεριφορά τους πριν τους επιτραπεί να το εφαρμόσουν στις προτεινόμενες από αυτές λύση. Για παράδειγμα, έστω ότι οι μαθητές γράφουν μια λειτουργία για να βρουν το μεγαλύτερο γειτονικό ζευγάρι αριθμών σε μια λίστα. Πριν να τους επιτραπεί η χρήση των δοκιμών που σχετίζονται με αυτό το πρόβλημα, οι μαθητές θα πρέπει πρώτα να επιλέξουν τη σωστή απάντηση στο ερώτημα "Ποιό είναι το αποτέλεσμα της κλήσης `largestPair(4, 3, -1, 5, 3, 3)`;" (Η σωστή απάντηση είναι «`(5, 3)`.) Σε ένα πανεπιστημιακό μάθημα 1300 ατόμων, η μεγάλη πλειοψηφία των μαθητών επέλεξε να επικυρώσει την κατανόησή των δοκιμαστικών τέστ με αυτόν τον τρόπο πριν να επιχειρήσουν να λύσουν τα προβλήματα, με αποτέλεσμα στη συνέχεια να κάνουν λιγότερες ερωτήσεις και να εκφράσουν λιγότερη σύγχυση σχετικά με το περιεχόμενο των προβλημάτων.
+
+Είναι πράγματι πολύ δελεαστικό να χρησιμοποιήσει κανείς τα αυτοματοποιημένα εργαλεία ελέγχου για τη βαθμολόγηση του κώδικα των μαθητών. Ωστόσο, [[Nutb2016](#CITE)] αρχικά δεν βρήκε συσχετισμό μεταξύ των βαθμών που παρέχονται από ανθρώπινη βαθμολόγηση και των παραβιάσεων που προέρχονται από τα αυτόματα εργαλεία. Μερικές φορές αυτό οφείλεται στο γεγονός ότι οι μαθητές παραβίασαν ένα μόνο κανόνα πολλές φορές (χάνοντας έτσι περισσότερους βαθμούς από ό,τι θα έπρεπε) ενώ άλλες φορές αυτό οφείλεται επειδή υπέβαλαν τον κώδικα εκφώνησης με λίγες μόνο αλλαγές και επομένως πήραν μεγαλύτερο βαθμό από όσο θα έπρεπε.
```

Comment:
[[Nutb2016](#CITE)]--> οι [[Nutb2016](#CITE)] αρχικά δεν βρήκαν
οφείλεται-->οφειλόταν

Reply:
Τhanks @pdelias ! Fixed in a new commit.